jetson/odrive_bridge/src/__main__.py

```diff
@@ -116,74 +116,74 @@ def __str__(self):
         return self.__class__.__name__
 
 
-class DisconnectedState(State):
-    def on_event(self, event):
+class Disconnected_State(State):

```

Comment:
see above, plz *dont* use the camel snake case mix

Reply:
sure